sklearn/preprocessing/tests/test_encoders.py

```diff
@@ -637,3 +637,32 @@ def test_categories(density, drop):
 @pytest.mark.parametrize('Encoder', [OneHotEncoder, OrdinalEncoder])
 def test_encoders_has_categorical_tags(Encoder):
     assert 'categorical' in Encoder()._get_tags()['X_types']
+
+
+@pytest.mark.parametrize('as_data_frame', ['True', 'False'])
```

Comment:
You can reuse the following code:

https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/utils/tests/test_utils.py#L239-L273

This is probably to move this code in `_testing.py` since it can be reuse in several location.
Then, you parametrize using different type of array and remove the for loop in the tests

Reply:
Thanks for the pointer.